### PR TITLE
use current locale to create the content type logs

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: LogBundle/EventSubscriber/LogContentTypeSubscriber.php [hunk 1]
```diff
@@ -11,6 +13,18 @@
...
+    protected $context;
+
+    /**
+     * @param Logger         $logger
+     * @param ContextManager $context
+     */
+    public function __construct(Logger $logger, ContextManager $context)
+    {
+        parent::__construct($logger);
+        $this->context = $context;
```
Comment (itkg-alavieille) [inline LogBundle/EventSubscriber/LogContentTypeSubscriber.php]: where this context is inject in service ?

**Comment by nicolasThal** [comment]
https://github.com/open-orchestra/open-orchestra-model-interface/pull/135
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/474
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1234

File: LogBundle/EventSubscriber/LogContentTypeSubscriber.php (hunk 1 above)
Comment (nicolasThal) [inline LogBundle/EventSubscriber/LogContentTypeSubscriber.php]: I have added it :)

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**